infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryPlan.kt

```diff
@@ -72,7 +72,7 @@ data class GeometryPlan(
     val planTime: Instant?,
     val uploadTime: Instant?,
     val units: GeometryUnits,
-    val trackNumberId: IntId<TrackLayoutTrackNumber>?,
+    val trackNumber: TrackNumber?,
```

Comment:
Ei tämän reviewin asiaa mutta tälle header vs plan erottelulle pitäisi kyllä tehdä jotain. On hassua tuplata valtaosa kentistä näin ja vaikka osalla kentistä on eri tyyppejä jäissain kohdissa, en oikein näe syytä että pitäisi olla. Esim plan voisi koostua headerista + muista.